Documentation/Configuration/Typo3ConfVars/EXT.rst

```diff
@@ -4,18 +4,38 @@
    TYPO3_CONF_VARS; EXT
 .. _typo3ConfVars_ext:
 
-==================================
-$GLOBALS['TYPO3_CONF_VARS']['EXT']
-==================================
+=====================================
+EXT - Extension manager configuration
+=====================================
+
+The following configuration variables can be used to configure settings for
+the Extension manager:
+
+..  contents::
+    :local:
+
+..  attention::
+    Extension specific configuration should be stored in
+    :php:`$GLOBALS['TYPO3_CONF_VARS']['EXTENSION']` and not here.
```

Comment:
I think this is not very clear, even I never know exactly what is meant by these words

Reply:
Which words are you talking about?